src/main/java/com/ciandt/techgallery/persistence/dao/impl/TechnologyRecommendationDAOImpl.java

```diff
@@ -84,4 +85,19 @@ public TechnologyRecommendation findByComment(TechnologyComment comment) {
     }
   }
 
+  @Override
+  public String findAllRecommendationsIdsStartingFrom(Date date) {
+    final Objectify objectify = OfyService.ofy();
+    final List<TechnologyRecommendation> allRecommendations =
+        objectify.load().type(TechnologyRecommendation.class).list();
```

Comment:
As we talked, we defined that TechnologyRecommendation must have a timestamp as well.